src/prost/build.rs

```diff
@@ -49,7 +59,13 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .collect();
 
     // Build protobuf structs.
-    let out_dir = PathBuf::from("./src");
+
+    // We first put generated files to `OUT_DIR`, then copy them to `/src` only if they are changed.
+    // This is to avoid unexpected recompilation due to the timestamps of generated files to be
+    // changed by different kinds of builds. See https://github.com/risingwavelabs/risingwave/issues/11449 for details.
+    //
+    // Put generated files in /src may also benefit IDEs https://github.com/risingwavelabs/risingwave/pull/2581
```

Comment:
Does CLion perform better today? 😄 IIRC, it also utilizes rust-analyzer. cc @Little-Wallace @liurenjie1024 👀

Reply:
I think intellij uses intellij-rust, written by same author of rust-analyzer.